install.sh

```diff
@@ -1284,8 +1282,8 @@ DB_PASS='$db_pass'
 DB_NAME='$db_name'"
 
 # remove ending forward slash for conf
-patchmgr=$(echo $relative_path|sed 's=/[^/]*$==;s/\.$//')
-targetdir=$(echo $new_web_dir|sed 's=/[^/]*$==;s/\.$//')
+patchmgr=${relative_path%/}
```

Comment:
this was not critical as there was made sure it always has a trailing slash. but if it had not, it became seriously crippled.

/a/b -> /a
/a/b/ -> /a/b